packages/lab/src/stepper-input/useStepperInput.ts

```diff
@@ -36,6 +40,10 @@ const toFloat = (inputValue: number | string) => {
 const sanitizedInput = (numberString: string) =>
   (numberString.match(ACCEPT_INPUT) || []).join("");
 
+const isIntermediateInput = (numberString: string) => {
```

Comment:
Is this covered in test?

Reply:
added test "does not call the `onChange` callback when the value is intermediate"